src/components/Editor/DrawingBoard/Canvas/Worker/EraserWorker.ts

```diff
@@ -94,6 +113,15 @@ class EraserWorker implements Worker {
       this.emit('renderAll', root.shapes);
     });
   }
+
+  erazerBox(point: Point): Box {
```

Comment:
Is `erazerBox` a typo? Or does it mean something else?

Reply:
It's a typo ..!  😭